packages/agoric-cli/src/lib/bundles.js

```diff
@@ -11,7 +11,7 @@ import { ZipReader } from '@endo/zip';
 /** @import {Bundle} from '@agoric/swingset-vat'; */
 /** @import {CoreEvalPlan} from '@agoric/deploy-script-support/src/writeCoreEvalParts.js' */
 
-const PACKAGE_NAME_RE = /(?<packageName>.*-v[\d.]+(?:-n\d+)?)\//;
```

Comment:
I find the named capture groups more clear but won't block.

Consider a regression unit test for different name strings observed that have failed.

Reply:
> Consider a regression unit test for different name strings observed that have failed.

That's what I came back here to add. I'd be willing to add a commit for that if it would help.

Reply:
I added some tests.  Feel free to add or subtract.